force-app/main/default/flows/Before_Save_Session_Registration_Validations.flow-meta.xml

```diff
@@ -0,0 +1,222 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Flow xmlns="http://soap.sforce.com/2006/04/metadata">
+    <apiVersion>54.0</apiVersion>
+    <assignments>
+        <name>Assign_JuncitonID_0</name>
+        <label>Assign JuncitonID</label>
+        <locationX>50</locationX>
+        <locationY>1463</locationY>
+        <assignmentItems>
+            <assignToReference>$Record.JunctionID__c</assignToReference>
+            <operator>Assign</operator>
+            <value>
+                <elementReference>CreateID</elementReference>
+            </value>
+        </assignmentItems>
+    </assignments>
+    <assignments>
+        <name>Attendee_Registered</name>
+        <label>Attendee Registered</label>
+        <locationX>50</locationX>
+        <locationY>551</locationY>
+        <assignmentItems>
+            <assignToReference>$Record.IsEventRegistered__c</assignToReference>
+            <operator>Assign</operator>
+            <value>
+                <booleanValue>true</booleanValue>
+            </value>
+        </assignmentItems>
+        <connector>
+            <targetReference>Get_Session_Registrations</targetReference>
+        </connector>
+    </assignments>
+    <assignments>
+        <name>Set_Overlapsed</name>
+        <label>Set Overlapsed</label>
+        <locationX>138</locationX>
+        <locationY>1031</locationY>
+        <assignmentItems>
+            <assignToReference>$Record.IsOverlaps__c</assignToReference>
+            <operator>Assign</operator>
+            <value>
+                <booleanValue>true</booleanValue>
+            </value>
+        </assignmentItems>
+        <assignmentItems>
+            <assignToReference>$Record.JunctionID__c</assignToReference>
```

Comment:
@bmtemakina you dont need to add junction assignment here, there are another assignment for the junction Id at the end of the flow.